src/wallet/rpcwallet.cpp

```diff
@@ -3841,7 +3841,7 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
         ret.pushKV("timestamp", meta->nCreateTime);
         if (!meta->hdKeypath.empty()) {
             ret.pushKV("hdkeypath", meta->hdKeypath);
-            ret.pushKV("hdmasterkeyid", meta->hdMasterKeyID.GetHex());
```

Comment:
E.g. here it seems that you are removing `hdmasterkeyid` from the dict. To make it easier for people that rely on this key to upgrade/downgrade without having to change their code (or configuration) every time they switch, it could make sense to leave the `ret.pushKV("hdmasterkeyid", meta->hdMasterKeyID.GetHex());` in for now. (Can be removed in the next release, for example)

Reply:
Ah, I see. And also have `hdseedid` return the same value? Yes, I can do that.

I can also hide `hdmasterkeyid` behind a deprecation switch if you think that's useful.

Reply:
Not sure if we need the overkill of a deprecation switch for every tiny api change. Just returning both and mentioning that one of the is deprecated in the docs seems pragmatic.

Reply:
Sounds good. I'll just return both and mention in the release notes

The deprecation switch isn't too heavy, so I can add it if anyone thinks it's necessary I can add it.